Makefile

```diff
@@ -123,20 +127,25 @@ public/favicon.ico: assets/images/logo.svg
 	convert assets/images/logo-16.png assets/images/logo-32.png assets/images/logo-64.png assets/images/logo-128.png -background white -alpha remove public/favicon.ico
 	rm assets/images/logo-128.png assets/images/logo-32.png assets/images/logo-64.png
 
+# builds a Docker container suitable to run the testsuite in
+# note: The container is called `.../openqa`.
 .PHONY: docker-test-build
 docker-test-build:
 	docker build --no-cache $(current_dir)/docker/openqa -t $(DOCKER_IMG)
 
+# prepares the environment variables for running tests within the Docker container
 .PHONY: docker.env
 docker.env:
 	env | grep -E 'FULLSTACK|UITEST|GH|TRAVIS|CPAN|DEBUG|ZYPPER' > $(docker_env_file)
 
+# launches a Docker container and runs the testsuite within, does NOT ensure that container is built
 .PHONY: docker-test-run
 docker-test-run: docker.env
 	docker run --env-file $(docker_env_file) -v $(current_dir):/opt/openqa -v /var/run/dbus:/var/run/dbus \
 	   $(DOCKER_IMG) make travis-codecov
 	rm $(docker_env_file)
 
+# launches a Docker container and runs the testsuite within (used on Travis)
```

Comment:
that's just the same as above again!!!1 I would ditch the commend and name it explicitly, e.g. "container-test-build-and-run"